src/frontend/src/components/CountingCard/styles.module.scss

```
@@ -0,0 +1,38 @@
+@use "../../styles/variables" as v;
+
+.container {
+  background-image: url(/background-small-card-3.png);
+  width: 327px;
+  height: 365px;
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.container__text {
+  text-align: center;
+  font-family: "Drive Corps", sans-serif;
+  font-size: 2rem;
+  font-style: normal;
+  font-weight: 400;
+  margin-bottom: 20px;
```

Alterar a medida para `rem`, evitando problemas de responsividade:
```suggestion
  margin-bottom: 1.25rem;
```